data/core/encyclopedia/geography.cfg

```diff
@@ -117,69 +149,89 @@ Over the River Aethen, south of Fort Tahn, is a Wesnothian frontier region. It i
-
-<header>text='Notable cities:'</header>
-    • Glamdrol: An Orcish tribal capital.
-    • Wesmere: The location of the Ka’lian — the Elvish Council.
-    • Dwarven Doors: A mixed human/dwarven town in the region of Knalga in the southern Heart Mountains. A major trade center.
-    • Dallben and Delwyn: Human villages originally built by settlers who crossed the Great River during Wesnoth’s Golden Age expansion. Now abandoned. The forested area northeast of <ref>dst='elensefar' text='Elensefar'</ref>, where these villages were located, was named the Annuvin province by men but was known by the elves as Wesmere.
-
-<header>text='Notable land features:'</header>
-    • <ref>dst='heart_mountains' text='Heart Mountains'</ref>: A virtually impassable barrier between the river country and the Northern Plains.
-    • Heartfangs: the particularly forbidding stretch of high peaks southwest of Lake Vrug and north of the Forest of Wesmere. The most inhospitable and dangerous portion of the Heart Mountains; only hermits, madmen, and mages live there.
-    • Swamp of Dread: a very large bog located between the Heart Mountains and the Great River. A notoriously dangerous place.
-    • Lake Vrug: A large mountain lake whose river carves the only pathway through the Northern Mountains.
-    • <ref>dst='arkan_thoria' text='Arkan-thoria'</ref>: The river that comes out of Lake Vrug. This is the elvish name; among humans it is called Longlier.
-    • River Listra: The south-running tributary of the Great River into which the Arkan-thoria empties.
-    • Lintanir Forest: The southernmost portion of the Great Northern Forest, a gigantic wood whose eastern and northern boundaries are known only to the elves. Their capital, Elensiria, has only seldom been visited by humans.
-    • <ref>dst='great_river' text='Great River'</ref>: The origin of this river is somewhere in the east of the northern lands."
+    text= _ "The Northlands are filled with unstable clans in endless conflict. It is an unlawful and perilous place plagued by its cold, icy terrain and constant war. Various groups of orcs, dwarves, men, gryphons, ogres, and even elves fight over the region. The northern and eastern borders of the Northlands are not defined, the southern border is the <ref>dst='great_river' text='Great River'</ref>, and the western border is the <ref>dst='great_ocean' text='ocean'</ref>.
+
+    Yet despite the sheer wilderness above the Northlands, the subterranean Dwarvish Kingdoms of Knalga and Kal Kartha are exceptions to their chaotic surroudings, providing both heat and protection to their people. While they may often be in conflict with orcs and trolls, the heart of Dwarvish territory is considered relatively safe in comparison." + "
```

Comment:
Separate issue: I wish spotted this earlier but it's bothered me re-reading this paragraph with the last few po updates - for the start of the paragraph I would say 'despite the sheer wilderness above ground' because the next clause refers to the underground. In the context of a geographical topic and with the convention of north being 'up', I initially read this as meaning 'north of the Northlands'.

No need to change anything now, but something to note for the next prose revision, if you're planning any.